src/main/java/io/reactivex/internal/operators/flowable/FlowableFlatMap.java

```diff
@@ -482,6 +482,9 @@ void drainLoop() {
                                     Exceptions.throwIfFatal(ex);
                                     is.dispose();
                                     errs.addThrowable(ex);
+                                    if (!delayErrors) {
+                                        upstream.cancel();
```

Comment:
So this is bug fix, but what's the reason for renaming `s` to `upstream`?

It looks not important to revert but I guess minimizing fix PRs saves time for those who review them while going through changelog :)

Reply:
It reads nicer now.